src/main/java/me/totalfreedom/totalfreedommod/command/Command_clearchat.java

```diff
@@ -0,0 +1,33 @@
+package me.totalfreedom.totalfreedommod.command;
+
+import me.totalfreedom.totalfreedommod.rank.Rank;
+import me.totalfreedom.totalfreedommod.util.FUtil;
+
+import org.bukkit.*;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = Rank.SENIOR_ADMIN, source = SourceType.BOTH, blockHostConsole = true, aliases = "cc")
+@CommandParameters(description = "Clear the public chat.", usage = "/<command>")
```

Comment:
This would not be quite accurate, as it would clear all chats, both public and admin chat.